reference/commands/custom_commands/commands/cmd_settings.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+from conan.api.conan_api import ConanAPIV2
+from conans.cli.command import conan_command
+from conans.client.conf import get_default_settings_yml
+from conans.errors import ConanException
+from conans.model.settings import Settings
```

Comment:
This is a bit dangerous. We don't know if we will make the ``Settings`` model public in the API. 
Same with other things like ``get_default_settings_yml``.

In theory, only ``conan.api`` and ``conans.cli`` (we need to move it to ``conan.cli``) should be used (maybe also ``conans.error`` (this is already available, make sure to use the ``conan.errors`` one)